src/components/change-wallet/WalletList.tsx

```diff
@@ -203,65 +226,64 @@ export default function WalletList({
-  }));
-
   const renderItem = useCallback(
     ({ item }) => {
       switch (item.rowType) {
         case RowTypes.ADDRESS:
           return (
             <Column height={item.height}>
               <AddressRow
+                contextMenuActions={contextMenuActions}
                 data={item}
                 editMode={editMode}
-                getEditMenuItems={getEditMenuItems}
-                getOnMenuItemPress={getOnMenuItemPress}
                 onPress={item.onPress}
-                watchOnly={watchOnly}
               />
             </Column>
           );
         default:
           return null;
       }
     },
-    [editMode, getEditMenuItems, getOnMenuItemPress, watchOnly]
+    [contextMenuActions, editMode]
   );
 
   return (
     <Container height={height}>
-      <Animated.View style={[StyleSheet.absoluteFill, emptyOpacityStyle]}>
-        <EmptyWalletList />
-      </Animated.View>
-      <WalletsContainer style={opacityStyle}>
-        <WalletFlatList
-          data={rows}
-          initialNumToRender={rows.length}
-          ref={scrollView}
-          renderItem={renderItem}
-          scrollEnabled={scrollEnabled}
-          showDividers={showDividers}
-        />
-        {showDividers && <WalletListDivider />}
-        {!watchOnly && (
-          <WalletListFooter>
-            <WalletOption
-              editMode={editMode}
-              icon="arrowBack"
-              label={`􀁍 ${lang.t('wallet.action.create_new')}`}
-              onPress={onPressAddAccount}
-            />
-            <WalletOption
-              editMode={editMode}
-              icon="arrowBack"
-              label={`􀂍 ${lang.t('wallet.action.add_existing')}`}
-              onPress={onPressImportSeedPhrase}
-            />
-          </WalletListFooter>
-        )}
-      </WalletsContainer>
+      {ready ? (
+        <WalletsContainer style={opacityStyle}>
+          <WalletFlatList
+            data={rows}
+            initialNumToRender={rows.length}
+            ref={scrollView}
+            renderItem={renderItem}
+            scrollEnabled={scrollEnabled}
+            showDividers={showDividers}
+          />
+          {showDividers && <WalletListDivider />}
+          {!watchOnly && (
+            <WalletListFooter>
+              <WalletOption
+                editMode={editMode}
+                label={`􀁍 ${lang.t('wallet.action.create_new')}`}
+                onPress={onPressAddAccount}
+              />
+              <WalletOption
+                editMode={editMode}
+                label={`􀂍 ${lang.t('wallet.action.add_existing')}`}
+                onPress={onPressImportSeedPhrase}
+              />
+            </WalletListFooter>
+          )}
+        </WalletsContainer>
+      ) : (
+        <Animated.View
+          exiting={FadeOut.easing(Easing.out(Easing.ease)).duration(
+            transitionDuration
+          )}
+        >
+          <EmptyWalletList />
+        </Animated.View>
```

Comment:
Could you verify with current develop version whether the layout animation wasn't removed?
IIRC I was removing it because it was causing crashes. Since layout animation don't work really well right now, we agreed not to use them. If you explicitly wanted to add it and you coded it, could you please try finding a way to implement it without using a layout animation?
https://www.notion.so/rainbowdotme/Reanimated-best-practices-5062be18b0e34594bb116b35bc43c5cb

Reply:
please, don't use layout animation

Reply:
`Animated.View` is still used in `WalletList.tsx` on develop, should I remove these usages too?